packages/react/runtime/types/types.d.ts

```diff
@@ -260,7 +260,15 @@ declare module '@lynx-js/types/background' {
 
   interface Performance {
     _generatePipelineOptions?(): PipelineOptions;
+    /**
+     * @since 3.1
+     */
     _onPipelineStart?(pipelineID: string, options: PipelineOptions): void;
+    /**
+     * @disabled since 3.1
+     * This version is disabled starting from version 3.1 due to additional required parameters.
+     */
+    _onPipelineStart?(pipelineID: string): void;
```

Comment:
I don't think we need to use override here. Use optional parameter instead.

```suggestion
    _onPipelineStart?(pipelineID: string, options?: PipelineOptions): void;
```

Reply:
Done. override function has been removed.